src/languages/java.ts

```diff
@@ -53,6 +53,11 @@ const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
   anonymousFunction: "lambda_expression",
   list: "array_initializer",
   functionCall: "method_invocation",
+  functionCallee: [
+    "method_invocation.identifier!",
+    "method_invocation[name]",
+    "object_creation_expression.type_identifier!",
```

Comment:
as above, should be
```suggestion
    "object_creation_expression[type]",
```

and worth adding tests to hit at least a few of the cases in https://github.com/tree-sitter/tree-sitter-java/blob/39a11c8330d0b2ad75310456c378396785f79a4e/grammar.js#L1007-L1015

Reply:
To be honest I don't know how to interpret this such that I could build test cases from it.

Reply:
@pokey do you mind talking me through how to interpret grammar.js for the sake of creating tests sometime?

Reply:
Did you mean to reference the tests in the repository under tests/corpus? I'm realizing now I probably should have referenced that. Or did you actually mean the grammar.js file?